DeploymentCloud/Deployment.Common/deployResources.ps1

```diff
@@ -845,8 +854,20 @@ if($sparkCreation -eq 'y') {
 
     $tokens = Get-Tokens
 	if ($useDatabricks -eq 'n') {
+
+        $sparkTemplate = "Spark-Template.json"
+        $sparkParameter = "Spark-parameter.json"
+
+        $version = ($HDInsightVersion -split '\.')[0]
+        $version = [int]$version
+        if ($version -ge 4 -and $enableHDInsightAutoScaling -eq 'y') {
+            $sparkTemplate = "Spark-AutoScale-Template.json"
+            $sparkParameter = "Spark-AutoScale-parameter.json"
+        }      
+        Write-Host "sparkTempalte: '$sparkTemplate' ; sparkParameter: '$sparkParameter'"
+
 		Write-Host -ForegroundColor Green "Estimated time to complete: 20 mins"
-		Deploy-Resources -templateName "Spark-Template.json" -paramName "Spark-parameter.json" -templatePath $templatePath -tokens $tokens
+		Deploy-Resources -templateName  $sparkTemplate -paramName $sparkParameter -templatePath $templatePath -tokens $tokens
```

Comment:
indentation needs to be fixed

Reply:
"Estimated time to complete: 20 mins" is still correct for 4.0?

Reply:
I looked one which took 16 minutes.

Reply:
The indent should be fixed